env.js

```diff
@@ -78,7 +79,8 @@ module.exports = (function() {
 
   env.discovery = {
     // The host to connect to for discovery
-    host: config.fromEnvironment('DISCOVERY_HOST')
+    host: config.fromEnvironment('DISCOVERY_HOST'),
+    skipHakken: config.fromEnvironment('SKIP_HAKKEN', false)
```

Comment:
so by default we do NOT skip Hakken right? this is good as I believe we will continue using it for a while

Reply:
That's it even if I'm not really sure that hakken is really useful in docker-compose environment and on our EC2 environments where all services are resolved on localhost.